modules/DSS-END-07_workflow-end-to-end.yml

```diff
@@ -9,8 +9,8 @@ screens:
         <ul class="fa-ul">
           <li><i class="fa fa-check-square-o"></i> Section 1: Introducing Git and GitHub</li>
           <li><i class="fa fa-check-square-o"></i> Section 2: Getting Started with Collaboration</li>
-          <li><i class="fa fa-check-square-o"></i> Section 3: The Basic GitHub Workflow</li>
-          <li><i class="fa fa-check-square-o"></i> Section 4: Creating Your First Pull Request</li>
+          <li><i class="fa fa-check-square-o"></i> Section 3: The GitHub Flow pt. 1: Branching and Committing</li>
```

Comment:
@1point618 ok -seriously - I'm not going to comment on this anymore - I know you will do a find and replace in all of these docs!